.github/workflows/build.yml

```
@@ -50,3 +50,8 @@ jobs:
       with:
         name: ${{ matrix.os }}-${{ matrix.arch }}-${{ matrix.mode }}
         path: package/**
+    
+    - uses: actions/upload-artifact@v4
+      with:
+        name: ${{ matrix.os }}-${{ matrix.arch }}-${{ matrix.mode }}-pdb
+        path: package_pdb/**
```

Related to https://github.com/maximegmd/CyberEngineTweaks/pull/999/files#r1929751443, this can be done using globs, not separate dirs.

---

In RED4ext I did this because I wanted to compute the checksum of the ZIP files. But the PDBs are in the same directory as the DLLs.

Here you can change both upload artifacts as below:

```suggestion
    # Upload artifacts
    - uses: actions/upload-artifact@v4
      with:
        name: ${{ matrix.os }}-${{ matrix.arch }}-${{ matrix.mode }}
        path: |
          package/**
          !package/**/*.pdb

    - uses: actions/upload-artifact@v4
      with:
        name: ${{ matrix.os }}-${{ matrix.arch }}-${{ matrix.mode }}-pdb
        path: package/**/*.pdb
```

The only thing I'm not sure about, is if the directory structure will be kept, but I assume it will.

---

Oh, right - thanks for the info, I'll test that variant when I get to a PC again.

---

Yeah it does preserve PDB path, you're right - changed it

---

Fixed according to review